packages/nextjs/package.json

```diff
@@ -86,7 +86,7 @@
     "@types/react-dom": "*",
     "expect-type": "^0.15.0",
     "jest": "*",
-    "node-fetch-native": "^0.1.8",
+    "node-fetch-native": "1.1.1",
```

Comment:
This is updated because, the previous version does not implement the `Response.json` function. The dependency is only used for testing.